extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -4,10 +4,11 @@ define('ext.wikia.adEngine.video.googleIma', [
 	'ext.wikia.adEngine.video.googleImaAdStatus',
 	'ext.wikia.adEngine.video.player.ui.closeButton',
 	'ext.wikia.adEngine.video.volumeControlHandler',
+	'wikia.browserDetect',
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (scriptLoader, googleImaAdStatus, closeButton, volumeControlHandler, doc, log, win) {
+], function (scriptLoader, googleImaAdStatus, closeButton, volumeControlHandler, browserDetect, doc, log, win) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 8 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)